api/server/routes/endpoints/anthropic/initializeClient.js

```diff
@@ -16,7 +15,12 @@ const initializeClient = async ({ req, res }) => {
     key = await getUserKey({ userId: req.user.id, name: 'anthropic' });
   }
   let anthropicApiKey = isUserProvided ? key : ANTHROPIC_API_KEY;
-  const client = new AnthropicClient(anthropicApiKey, { req, res });
+  if (typeof ANTHROPIC_REVERSE_PROXY === 'undefined') {
+    Reverse_Proxy = undefined;
```

Comment:
you should not be using a global variable

Reply:
my bad one sec lemme use let instead gonna fix comments aswell